src/main/java/com/limechain/rpc/methods/author/AuthorRPCImpl.java

```diff
@@ -0,0 +1,169 @@
+package com.limechain.rpc.methods.author;
+
+import com.limechain.rpc.methods.author.dto.DecodedKey;
+import com.limechain.rpc.methods.author.dto.DecodedKeysReader;
+import com.limechain.runtime.RuntimeEndpoint;
+import com.limechain.storage.block.BlockState;
+import com.limechain.storage.crypto.KeyStore;
+import com.limechain.storage.crypto.KeyType;
+import com.limechain.utils.StringUtils;
+import com.limechain.utils.scale.ScaleUtils;
+import io.emeraldpay.polkaj.scale.ScaleCodecWriter;
+import io.emeraldpay.polkaj.schnorrkel.Schnorrkel;
+import io.emeraldpay.polkaj.schnorrkel.SchnorrkelException;
+import io.emeraldpay.polkaj.schnorrkel.SchnorrkelNative;
+import io.libp2p.crypto.keys.Ed25519PrivateKey;
+import org.bouncycastle.crypto.params.Ed25519PrivateKeyParameters;
+import org.javatuples.Pair;
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Service
+public class AuthorRPCImpl {
+
+    private final BlockState blockState = BlockState.getInstance();
```

Comment:
I suggest we set all beans and singletons inside of class constructors. It might be a good guideline to start following. Wdyt?